dataset/dataset.go

```diff
@@ -48,6 +49,20 @@ type Client struct {
 	url string
 }
 
+// closeResponseBodyWithContext closes the response body and logs an error containing the context if unsuccessful
+func closeResponseBodyWithContext(ctx context.Context, resp *http.Response) {
+	if err := resp.Body.Close(); err != nil {
+		log.ErrorCtx(ctx, err, log.Data{"Message": "error closing http response body"})
```

Comment:
I believe logging standards say the keys in log.Data should be lower cased. Not sure whether the aggregator is case sensitive 🤔

Reply:
Haha, in which case this has been a mistake on all client updates.